src/components/Toggle/ToggleSizes.tsx

```diff
@@ -0,0 +1,4 @@
+export enum ToggleSizes {
+  Large = "lg",
```

Comment:
To follow the same patterns as other components, the string values should not be abbreviated.  `Icon` and `Image` are two components that use the patterns that I am recommending.
```
export enum ToggleSizes {
  Large = "large",
  Small = "small",
}
```

Reply:
Because "lg" and "sm" are specifically the values expected by Chakra, I added this: https://github.com/NYPL/nypl-design-system/pull/758/files#diff-7ae8cfe58c40cccbd1d760ad907b63d018daeebebb15d0c63a33a8d15c38cfbcR91 which I'm pointing out in case there's a specific pattern we want to follow

Reply:
I'm okay with this. I think this may be the first time we use Chakra's own `size` prop to set the specific size we want.

Reply:
If we can rely on the typescript enums, then yes, this is fine.  But what about scenarios where the enums are not being used?  Then our pattern changes.